ui/app/templates/vault/cluster/mfa-setup.hbs

```diff
@@ -8,28 +8,26 @@
     <h1 class="title is-4">MFA setup</h1>
   </:header>
   <:content>
-    <div class="auth-form" data-test-mfa-form>
-      <div class="box">
-        {{#if (eq this.onStep 1)}}
-          <Mfa::MfaSetupStepOne
-            @isUUIDVerified={{this.isUUIDVerified}}
-            @restartFlow={{this.restartFlow}}
-            @saveUUIDandQrCode={{this.saveUUIDandQrCode}}
-            @showWarning={{this.showWarning}}
-            data-test-step-one
-          />
-        {{/if}}
-        {{#if (eq this.onStep 2)}}
-          <Mfa::MfaSetupStepTwo
-            @entityId={{this.entityId}}
-            @uuid={{this.uuid}}
-            @qrCode={{this.qrCode}}
-            @restartFlow={{this.restartFlow}}
-            @warning={{this.warning}}
-            data-test-step-two
-          />
-        {{/if}}
-      </div>
-    </div>
+    <Hds::Card::Container @level="high" @hasBorder={{true}} class="has-padding-l" data-test-mfa-form>
```

Comment:
just wrapped in the `Hds::Card`
<img width="394" alt="Screenshot 2025-05-05 at 2 18 42 PM" src="https://github.com/user-attachments/assets/cf164749-f545-460a-bbe7-2f17afa876e7" />